### PR TITLE
Add software scientist posting

### PR DESCRIPTION
**Pull request by j-wags**
Also corrects funding source in postdoc posting, and adds HTML anchors.

### CODE REVIEW

File: content/jobs/_index.md [hunk 1]
```diff
@@ -55,17 +54,107 @@ Interested candidates should email an application to `info@openforcefield.org` w
...
+- Broadening our engagement with the open-source chemistry community 
+- Leading regular science subproject meetings and writing 
+  [blog posts](https://openforcefield.org/science/updates/propertycalculator-meeting-2019-10-17/) to 
+  summarize progress 
+- Representing Open Force Field at conferences and workshops 
+- Traveling to [project labs](https://openforcefield.org/members/) as needed for focused in-person work 
+- Debugging CI builds, dealing with conda packaging issues, and helping shepherd the eventual migration 
+  of omnia packages to conda-forge 
+
+Writing papers is _not_ a major focus of this position.
```
Comment (jchodera) [inline content/jobs/_index.md]: What if we point out that their effort would still be acknowledged with coauthorship in publications featuring their work, though leading the writing of papers or new scientific studies is not a major focus?

File: content/jobs/_index.md [hunk 2]
```diff
@@ -55,17 +54,107 @@ Interested candidates should email an application to `info@openforcefield.org` w
...
+
+The position includes up to 25% travel, and the software scientist will be assigned a PI that they will
```
Comment (jchodera) [inline content/jobs/_index.md]: Does "25% travel" mean that you will be working from somewhere besides your primary location 25% of the year? That's a lot!

Comment (j-wags) [reply content/jobs/_index.md]: Figured it was safest to cover our bases with a "worst case" statement, but many people have mentioned this now. I've dropped it to 15% (which is still 7 weeks in a year)

File: content/jobs/_index.md (hunk 1 above)
Comment (j-wags) [reply content/jobs/_index.md]: Added your explanation.

**Pull request merged**